nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# #17 Uploadimages

Fixes:
- user now able to upload images when creating a new product to sell and the image will display on the product details page
-

Changes proposed in this request & reasons behind organizational or architectural decisions:
-
-

Steps to test:
- create a product and upload an image from your files
- image should go into the images folder in the wwwroot folder
- when you click on the details tab on the index the image should be at the bottom of the page
-

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
- N/A
-

Commit message:
- user can now upload pictures to products

Link to feature ticket:
- #17 

@mighty-mahi-mahi
